Sources/ArcGISToolkit/Documentation.docc/Tutorials/JobManagerTutorial.tutorial

```diff
@@ -0,0 +1,138 @@
+            }
+            
+            @Step {
+                Eventually, the job will poll the service and receive the response that the work is complete and there is a result available to download. At this point the job starts a URL session and downloads the result.
+                
+                The mobile map package is downloaded from the server to a local folder.
+                
+                @Image(source: JobManagerTutorialJobDownload, alt: "An image of the app UI with a progress bar one hundred percent done.")
+            }
+            
+            @Step {
+                To generate the offline map you will need an API key or credential. This step show how to add an API key. To use a credential refer to the AuthenticatorTutorial.
+                
+                Once an API key has been added the project can be built and run. Tap the "Take Map Offline" button to add the job to the job manager and start the job. The offline map will be generated and downloaded. As long as the tutorial app is running in the foreground the job will finish and we'll get the mobile map package. However, we know that people don't use their phones like that. What we want is for people to be able to start a job, do other things on their phone while the job runs and get a notification when the job finishes. In the next steps you will add the code to handle the app being backgrounded or terminated.
+                
+                @Code(name: "JobManagerTutorialApp.swift", file: JobManagerAppStep1)
+            }
+            
+            @Step {
+                Adding background tasks begins with some project setting. Enable the kind of background processing needed by the app in the Background Modes. The job manager uses "Background fetch".
+                
+                The job manager registers a background task to allow all of it's jobs to check the status of the work happening on the servers. Getting the status is a lightweight operation that requires "Background fetch" mode.
+                
+                @Image(source: JobManagerTutorialBackgroundFetch, alt: "An image of the Background Modes settings with the Background fetch mode checked.")
+            }
+            
+            @Step {
+                Now tell the background task scheduler the identifier of the task that we want to run in the background. This is done in the project info for the app target. Add "com.esri.ArcGISToolkit.jobManager.statusCheck" as the first item in the Permitted Background Task Scheduler Identifiers.
+                
+                The job manager will register a background task with this identifier and submit a request to have some background time periodically. In this tutorial we will ask to be given time every 30 seconds. When the app gets background time is at the operating system's disgression; it may or may not be anywhere close to 30 seconds.
```

Comment:
Does this read a bit clearer?

```suggestion
                The job manager will register a background task with this identifier and submit a request to be allotted periodic background processing time. In this tutorial we will ask to be given time every 30 seconds. When the app gets background time is at the operating system's digression; it may or may not be anywhere close to 30 seconds.
```